modules/generator/registry/interface.go

```diff
@@ -4,7 +4,7 @@ package registry
 type Registry interface {
 	NewLabelValueCombo(labels []string, values []string) *LabelValueCombo
 	NewCounter(name string) Counter
-	NewHistogram(name string, buckets []float64) Histogram
+	NewHistogram(name string, buckets []float64, histogramOverride string) Histogram
```

Comment:
Minor: can we make this an enum? I know enums are kind of weak in Go, but it helps document the interface what values are allowed.

Reply:
The diff is somewhat large.  Let's take this and I'll put up another for this request.

Reply:
https://github.com/grafana/tempo/pull/3938 to follow up.